.github/workflows/lint.yml

```diff
@@ -52,4 +52,17 @@ jobs:
         run: pip3 install pyright
       - name: Run pyright
         run: pyright tasks
-  
+  lint-python:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install lint python requirements
+        run: pip3 install -r https://raw.githubusercontent.com/DataDog/datadog-agent-buildimages/main/circleci/requirements.txt
```

Comment:
Local requirements.txt might be enough

Reply:
I used this file because it's the one that lists the required linters used in datadog-agent, it will allows remaining aligned in terms of configuration, wdyt?